.golangci.yml

```diff
@@ -41,7 +41,7 @@ linters:
     # - godot
     - gofmt
     # - gofumpt
-    - goheader
+    # goheader
```

Comment:
Well, disabling a linter is hardly fixing the issue, is it? 🙃 We need to change how the year is checked in the header [template](https://github.com/conduitio-labs/conduit-connector-marketo/blob/0e497bc33ca7e1ff0886b8c607b32d86a1529daa/.golangci.goheader.template#L1). In other repos we use a custom tag ([example](https://github.com/ConduitIO/conduit-connector-kafka/blob/15bc7f969f19af0457d4cc6ebc94a270ee8fabf4/.golangci.goheader.template#L1)) and define a regex for the copyright year ([example](https://github.com/ConduitIO/conduit-connector-kafka/blob/15bc7f969f19af0457d4cc6ebc94a270ee8fabf4/.golangci.yml#L18)).